java-checks/src/main/java/org/sonar/java/checks/helpers/ExpressionsHelper.java

```diff
@@ -85,32 +86,32 @@ public static Tree reportOnClassTree(ClassTree classTree) {
 
 
   public static ValueResolution getConstantValue(ExpressionTree expression) {
```

Comment:
`getConstantValue` and `getConstantValueAsInteger` are not used, and not tested, we might want to remove them.